app/services/npq/application/change_funded_place.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module NPQ
+  module Application
+    class ChangeFundedPlace
+      include ActiveModel::Model
+      include ActiveModel::Attributes
+
+      attribute :npq_application
+      attribute :funded_place
+
+      validates :npq_application, presence: { message: I18n.t("npq_application.missing_npq_application") }
+      validate :accepted_application
+      validate :eligible_for_funding
+      validate :eligible_for_removing_funding_place
+      # validate :funding_cap_available
+
+      def call
+        return self unless valid?
+
+        ApplicationRecord.transaction do
+          if FeatureFlag.active?("npq_capping")
+            npq_application.update!(funded_place:)
+          end
+        end
+
+        npq_application
+      end
+
+    private
+
+      def accepted_application
+        return if npq_application.accepted?
+
+        errors.add(:npq_application, I18n.t("npq_application.cannot_change_funded_status_from_non_accepted"))
+      end
+
+      def eligible_for_funding
+        return if npq_application.eligible_for_funding?
+
+        errors.add(:npq_application, I18n.t("npq_application.cannot_change_funded_status_non_eligible"))
+      end
+
+      def eligible_for_removing_funding_place
+        return if npq_application.funded_place?
+
+        declarations_states = npq_application
```

Comment:
maybe we can try something like applicable declarations? 
```
participant_profile.participant_declarations
      .where(state: %w[submitted eligible payable paid])
```
and returning one error? see change schedule [service](https://github.com/DFE-Digital/early-careers-framework/blob/main/app/services/change_schedule.rb#L209) as an example